generate/src/contract/methods.rs

```diff
@@ -65,8 +69,7 @@ fn expand_functions(cx: &Context) -> Result<TokenStream> {
         }
 
         /// Type containing all contract methods for generated contract type.
-        #[allow(non_camel_case_types)]
-        #[derive(Clone)]
+        #methods_attrs
```

Comment:
The non-camel case is no longer needed?

Reply:
No, it was needed before #242, and was left there by accidednt.